packages/react-components/src/components/xy-output-overview-component.tsx

```diff
@@ -334,4 +342,15 @@ export class XYOutputOverviewComponent extends AbstractXYOutputComponent<Abstrac
     protected getTitleBarTooltip(): string {
         return this.props.outputDescriptor.name + ' overview';
     }
+
+    protected showOptionsMenu(): React.ReactNode {
+        return <React.Fragment>
+            {<div className='options-menu-container'>
+                <button title="Select overview output source" className='options-menu-button'
+                onClick={()=>{signalManager().fireSelectOverviewOutputSignal(this.props.experiment);}}>
+                    <FontAwesomeIcon icon={faCog} />
```

Comment:
I wonder if we should have the "hamburger menu" instead and have the config as an entry in the menu. Not sure though. What do you think?

Reply:
True and I was aware of that.  In the future we might have more options for all the views. I want to avoid to have multiple buttons in the views, because of limited space. Other views (xy or timegrah) have the hamburger menu with only the "Pin view" in it at the moment. I'll get back to you about this.